Libraries/Animated/src/Interpolation.js

```diff
@@ -163,6 +165,16 @@ function interpolate(
   return result;
 }
 
+function colorToRgba(
+  input: string
+): string {
+  var color = tinycolor(input);
+  var {r, g, b, a} = color.toRgb();
```

Comment:
Not safe? Tinycolor default to black, so it won't crash!
What do you mean by not safe?
(I'll change it anyway ^^)